### PR TITLE
Extend parse's implementation to accept both formats

### PR DESCRIPTION
**Pull request by amatgil**

### CODE REVIEW

**Comment by kaikalii** [comment]
Can you please add tests to `tests/units.rs`?

**Comment by amatgil** [comment]
Certainly
(Though a type of each case is in the documentation and gets run as a test already)

File: tests/units.ua [hunk 1]
```diff
@@ -211,6 +211,8 @@
 ⍤⤙≍ [2 10 5] ⬚5⋕{"2" "10" "dog"}
 ⍤⤙≍ [ℂ 3.4e6 1.2 ℂ π ∞] ⋕ {"1.2r3.4e6i" "infrpii"}
 ⍤⟜≍ ⋕°⋕. [ℂ 3.4e6 1.2 ℂ π ∞]
+⍤⤙≍ [ℂ4.5 `12.3 ℂ4.5 12.3 ℂ`4.5 12.3 ℂ`4.5 12.3 ℂ4.5 12.3] ◇⋕ {"-12.3r4.5i" "12.3r4.5i" "12.3-4.5i" "12.3r-4.5i" "12.3r4.5i"}
```
Comment (kaikalii) [inline tests/units.ua:214]: Why `◇ content`?

Can you add some tests to verify that integers also work?
And `pi`, `inf` etc.

**Comment by amatgil** [comment]
I might've added too many tests, I can't tell what's too much and what's too little

**Pull request merged**